packages/dendron-cli/src/commands/podsV2.ts

```diff
@@ -0,0 +1,279 @@
+import {
+  DendronError,
+  DEngineClient,
+  ERROR_SEVERITY,
+  NoteProps,
+  NoteUtils,
+  RespV3,
+  URI,
+  VaultUtils,
+} from "@dendronhq/common-all";
+import {
+  ConfigFileUtils,
+  PodExportScope,
+  PodUtils,
+} from "@dendronhq/pods-core";
+import _ from "lodash";
+import path from "path";
+import yargs from "yargs";
+import { setupEngine, SetupEngineCLIOpts, SetupEngineResp } from "./utils";
+
+export type PodCLIOpts = {
+  podConfig?: URI;
+  inlineConfig?: string[];
+  configValues?: { [key: string]: any };
+  vault?: string;
+  fname?: string;
+  hierarchy?: string;
+  podId?: string;
+};
+
+export type PodCommandCLIOpts = {} & SetupEngineCLIOpts & PodCLIOpts;
+
+export type PodCommandOpts<T = any> = PodCLIOpts & {
+  config: T;
+  payload: NoteProps[];
+} & SetupEngineResp &
+  SetupEngineCLIOpts;
+
+export function setupPodArgs(args: yargs.Argv) {
+  args.option("inlineConfig", {
+    describe:
+      "pass in config instead of reading from file. format is Key={key},Value={value}. If provided, this will override the value saved in the config file",
+    array: true,
+  });
+  args.config(
+    "podConfig",
+    "*.yml configuration file for pod",
+    function (configPath) {
```

Comment:
nitpick: eslint warning about function expressions here, it recommends arrows instead